services/app/src/components/views/ExpansionCheckbox.test.tsx

```diff
@@ -0,0 +1,23 @@
+import {mount} from 'app/Testing';
+import * as React from 'react';
+
+import ExpansionCheckBox, { IProps } from './ExpansionCheckbox';
+
+describe('ExpansionCheckBox', () => {
+
+  function createElement() {
+    const props: IProps = {
+      onChange: jest.fn(),
+      contentSets: { horror: true },
+      value: undefined,
+    };
+    const elem = mount(<ExpansionCheckBox {...props} />);
+    return {elem, props};
+  }
+
+  test('changes search param values when onChange is clicked', () => {
```

Comment:
Can you also add a test to confirm the desired initial behavior based on settings? (i.e. that if the content set is enabled, it starts checked, and if it's disabled, it mentions settings in the text)